### PR TITLE
sklako.oksana

### PR DESCRIPTION
**Pull request by ok-skalko**

### CODE REVIEW

File: skalko.oksana/homework_8/notification.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  def add_to_log(recepient)
+    if instance_of?(Sms)
+      File.open('Sms.log', 'w') { |txt| txt.write(recepient) }
+    else
+      File.open('Email.log', 'w') { |txt| txt.write(recepient) }
+    end
+  end
+
+  def send_message(recepient)
```
Comment (alexgonor) [inline skalko.oksana/homework_8/notification.rb:19]: this method should run block code

File: skalko.oksana/homework_8/notification.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
+module Notification
+  VALID_NUMBER = /^\+?3?8?(0[5-9][0-9]\d{7})$/.freeze
+  VALID_EMAIL = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i.freeze
+
+  def self.included(base)
+    base.extend(ClassMethods)
+  end
+
+  module ClassMethods
+    def log
```
Comment (alexgonor) [inline skalko.oksana/homework_8/notification.rb:10]: `p File.read("#{name}.log")`
and you can remove all other code in this method

File: skalko.oksana/homework_8/notification.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      if name.eql? 'Sms'
+        p File.read('Sms.log')
+      else
+        p File.read('Email.log')
+      end
+    end
+  end
+
+  def add_to_log(recepient)
```
Comment (alexgonor) [inline skalko.oksana/homework_8/notification.rb:15]: `File.open("#{self.class}.log", 'w') { |txt| txt.write(recepient) }`
same as in my comment above

**Review by alexgonor — CHANGES_REQUESTED** [review]

File: skalko.oksana/homework_8/notification.rb (hunk 2 above)
Comment (ok-skalko) [reply skalko.oksana/homework_8/notification.rb:10]: ok, thanks

**Review by alexgonor — APPROVED** [review]

**Pull request merged**